config/defaults.ini

```diff
@@ -301,6 +301,16 @@ regular_users_can_manage_content = false
 #   # Kolibri 0.12.2 User Guide for Admins [document]
 #   5bb37c1832c8489ab2940f31588305f6
 
+[endlesskey]
+# Preloaded content or the Endless Key app
+# The Endless Key app can be pre-loaded with content from the Kolibri channels
+# listed in this setting.
+install_channels =
```

Comment:
Is it intentional for the moment that we're going to do the old content nodes exhaustively listed in the image configuration? Another option here is we could add an `install_content_collections` option which lists names of collection files from [endless-key-collections](https://github.com/endlessm/endless-key-collections). The good news about those is Kolibri's `importcontent` management command already knows how to read them. The bad news is it would involve a whole new hook (and probably a Python script because that management command is a smidge limited at the moment).